### PR TITLE
typo: power to powered

### PR DESCRIPTION
**Pull request by liaocp666**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

用 powered 更加合适

#### Screenshots:

image

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by ruibaby** [comment]
我觉得可以直接改为 Powered By Halo。

/cc @halo-dev/sig-halo-admin

File: src/components/GlobalFooter/GlobalFooter.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="footer text-center" style="padding: 0 16px; margin: 48px 0 0">
     <div class="copyright" style="color: rgba(0, 0, 0, 0.45); font-size: 14px">
-      Proudly power by
+      Proudly powered by
```
Comment (JohnNiang) [inline src/components/GlobalFooter/GlobalFooter.vue:4]:
```suggestion
      Powered by
```

**Review by ruibaby — COMMENTED** [review]
/approve

**Comment by ruibaby** [comment]
@liaocp666 请按照 https://github.com/halo-dev/halo-admin/pull/597#issuecomment-1214177515 签署一下我们的 CLA。

**Comment by liaocp666** [comment]
@ruibaby 👌

**Comment by liaocp666** [comment]
签署好了

**Review by guqing — COMMENTED** [review]
/lgtm

**Pull request merged**